src/java/org/apache/cassandra/cql3/statements/schema/AlterSchemaStatement.java

```diff
@@ -150,4 +152,15 @@ static InvalidRequestException ire(String format, Object... args)
     {
         return new InvalidRequestException(String.format(format, args));
     }
+
+    public static abstract class Raw<R extends AlterSchemaStatement> extends CQLStatement.Raw
+    {
+        @Override
+        public R prepare(ClientState state)
```

Comment:
would it make sense to make this method `final` here?

Anyway, I'd love some javadoc.